controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/CommitRequestHandler.java

```diff
@@ -162,9 +169,9 @@ public CommitRequestHandler(final StreamMetadataStore streamMetadataStore,
                                                         activeEpochRecord.getReferenceEpoch() == txnEpochRecord.getReferenceEpoch()) {
                                                     // If active epoch's reference is same as transaction epoch,
                                                     // we can commit transactions immediately
-                                                    return commitTransactions(scope, stream, activeEpochRecord.getSegments(), txnList, context);
+                                                    return commitTransactions(scope, stream, activeEpochRecord.getSegments(), txnList, context, requestId);
```

Comment:
I have seen `requestId` in the beginning, in the middle, and at the end of the argument list. What's the criteria we are using and can we be consistent about the positioning of the parameter?

Reply:
There are three aspects here:
- In all the (private) methods and classes where `requestId` has been added in this PR, it has been done at the end of the argument list (e.g., `AutoScaleTask:postScaleRequest`, `SegmentHelper`) / attribute list (e.g., `AutoScaleEvent`, `DeleteStreamEvent`).
- For `TagLogger`, we use `requestId` as the first parameter, in the same way that `Logger` uses markers (also as a first parameter).
- For legacy methods already having a `requestId` parameter, such as `PravegaRequestProcessor:handleException` or `PravegaRequestProcessor:verifyToken`, the legacy argument position has been respected.